pkg/apis/numaflow/v1alpha1/vertex_types.go

```diff
@@ -367,6 +367,9 @@ type AbstractVertex struct {
 	// More info: https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
 	// +optional
 	InitContainers []corev1.Container `json:"initContainers,omitempty" protobuf:"bytes,11,rep,name=initContainers"`
+	// List of sidecar containers belonging to the pod.
+	// +optional
+	SidecarContainers []corev1.Container `json:"sidecarContainers,omitempty" protobuf:"bytes,12,rep,name=sidecarContainers"`
```

Comment:
thoughts on naming? `sidecarContainers`, `containers`, `sidecars`?

Reply:
+1 for `sidecars`.